sandthorn_event_filter.gemspec

```diff
@@ -28,6 +28,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rake"
   spec.add_development_dependency "codeclimate-test-reporter"
 
-  spec.add_runtime_dependency "hamster"
+  spec.add_runtime_dependency "hamster", "=1.0.0"
```

Comment:
I dunno, I think they've respected semantic versioning earlier. 1.X should be okay. There are release candidates for `1.0.1`. I think we will want to follow patch releases.